src/api/api.ts

```diff
@@ -21,6 +30,13 @@ export async function getAssetList({
   return response.items;
 }
 
+export function fetch3DModelRevision(
```

Comment:
Any reason to have this wrapper?

Reply:
If we might reuse this ThreeD.retrieveRevision call it will use single realisation. In case of changing sdk call's signature it will be easier to fit changed sdk call in our app.